### ISSUE TITLE
[Android] - initWallet - descriptor not always defined

### ISSUE THREAD

**Issue by Czino** [LtbLightning/bdk-rn#75]
With the new PR, we encounter a new occasional error, it appears often only once or twice and then not again during the session.

Possibly a racing condition caused by wrapping every method in a Thread

https://github.com/LtbLightning/bdk-rn/blob/933cee711893e3644b33ef7925ca1996a9b2a0ea/android/src/main/java/io/ltbl/bdkrn/BdkRnModule.kt#L407-L428

**Comment by Czino** [comment]
The odd thing is, that functions such as `newBip84` return the descriptor id for use.

**Comment by BitcoinZavior** [comment]
Managed to replicate this issue, we should have a fix shortly 👍

**Comment by BitcoinZavior** [comment]
> Managed to replicate this issue, we should have a fix shortly 👍

PR Updated, please see if you still have the issue
